source/index.rst

```diff
@@ -34,14 +34,14 @@ converse.js chat client.
 Registering a new chat account
 ==============================
 
-.. figure:: images/register-panel.jpg
+.. figure:: images/register-panel.png
    :align: left
    :alt: The registration panel of the converse.js control box.
 
    The registration panel of converse.js
 
 With converse.js you can register a new XMPP account on any publically available XMPP provider.
-To do so, click the **Register** tab on the chat client.
+To do so, click the **Register** link to the bottom of panel.
```

Comment:
The link doesn't say **Register**, it says **Create an account**

Reply:
Also, it would be better to say "below the login form", instead of "to the bottom of the panel"